context/app/static/js/components/Routes/Routes.jsx

```diff
@@ -262,7 +262,7 @@ function Routes({ flaskData }) {
     const workspaceId = urlPath.split('/').pop();
     return (
       <Route>
-        <WorkspacePleaseWait workspaceId={workspaceId} />
+        <WorkspacePleaseWait workspaceId={parseInt(workspaceId, 10)} />
```

Comment:
Should we do the `parseInt` call on line 262 instead? I don't think it particularly matters but it would keep this a bit cleaner.